### DISCUSSION TITLE
How do I expand the Commit Graph?

### DISCUSSION THREAD

**Discussion by ikocs** [jesseduffield/lazygit#4128] [Q&A]
By default I don't see the graph itself, just a small circle. 
* Nerd Font is set. 
* Graph display through <c-l> enabled</c-l>.

![image](https://github.com/user-attachments/assets/93c36c7b-926c-4758-b6d7-e8468e1457a0)

**Comment by stefanhaller** [comment]
That _is_ the graph. Your branch is just a linear sequence of commits, so there is nothing else to show except a string of circles. (If you were looking for a connecting line between them, there's not enough room to show it in a terminal UI.)

**Reply by ikocs** [reply]
It turned out to be like that, thank you very much! But I noticed one strange thing (maybe it's my inattention). I changed the comit sorting mode to `date-order` and the branches appeared: 

![image](https://github.com/user-attachments/assets/90d3f627-f8fb-4a7c-b816-631f6744bd22) 

Then I tried to switch to the view on the screenshot above, but it didn't work. Although I switched back on sorting `topo-order`.

**Reply by stefanhaller** [reply]
It looks like you (maybe accidentally) turned on the "Toggle show whole git graph" setting in the Log menu. This will show other branches besides the current one. Be aware though that this feature is a bit broken (see #3608), so use this with care.
